### DISCUSSION TITLE
Anyone tried the v8 alpha, yet?

### DISCUSSION THREAD

**Discussion by Obiwarn** [TanStack/table#3764] [General]
The lack of Typescript support in v7 drives me crazy. 

Has anyone tried the v8 alpha, yet? The code looks quite complete.

**Comment by quantizor** [comment]
Not yet, but I have questions about it. @tannerlinsley is this the appropriate place for general questions about the upcoming rewrite?

I've been using `react-table-sticky` with some custom code to handle grid layout + stickiness. If the plugin system goes away, will this sort of functionality be supported in core?

In particular, I have code that allows for setting grid child values for things like `column.minWidth: "max-content"` and then post initial render I measure what the browser ended up with and replace the values so normal handling takes over...

**Reply by tannerlinsley** [reply]
Out of the box, plugins won't work with the new system because there isn't really a "plugin" system any more. However, I fully plan on supporting the right abstractions to allow dev to wrap `useTable` with whatever they want and be able to achieve the same results. 

Some of those abstraction points are already available, but many of them are definitely not and have yet to be discovered. The best way of doing this is by opening a new issue/PR with your plugin. Boil it down to the bare minimum functionality that you need covered and then we can discuss what it would take from the core to achieve the same result.

**Reply by tsingson** [reply]
i m try v8 with jotai ( a powerful state management ) , it's complete new design.
it's look great for typescript

**Reply by jlorezz** [reply]
> i m try v8 with jotai ( a powerful state management ) , it's complete new design. it's look great for typescript

What was your approach to this?

**Comment by seanonthenet** [comment]
Excited to try. Will do this week!

**Comment by colindjk** [comment]
@tannerlinsley Curious, in terms of code functionality how close to completion is the alpha version of v8? (Will there be a beta etc). 

Testing it out and it makes certain things easier for sure. Would be nice to know of any potential release timelines :). Thanks.

**Comment by tannerlinsley** [comment]
The readme has the todo list. When all of the core features and most of the examples are done, I’ll release a beta.

Tanner Linsley
